### PR TITLE
Issues/#4894 notifications periods

### PR DESCRIPTION
**Pull request by gcusnieux**
Related to gravitee-io/issues#4894

### CODE REVIEW

**Review by aelamrani — CHANGES_REQUESTED** [review]
Time is not well updated in the input.

Apr-30-2021 08-37-49

File: stories/lib/utils.test.js [hunk 1]
```diff
@@ -77,4 +77,16 @@ describe('U T I L S', () => {
   test('should compare null objects', () => {
     expect(deepEqual(null, null)).toEqual(true);
   });
+
+  test('should compare numbers arrays', () => {
+    expect(deepEqual([1, 2], [1, 2])).toEqual(true);
+  });
+
+  test('should compare string arrays', () => {
```
Comment (gaetanmaisse) [inline stories/lib/utils.test.js:85]: `string arrays` -> 🔍 in the test we are using number arrays

File: stories/lib/utils.test.js [hunk 2]
```diff
@@ -77,4 +77,16 @@ describe('U T I L S', () => {
...
+    expect(deepEqual([2, 1], [1, 2])).toEqual(false);
+  });
+
+  test('should compare string arrays without order', () => {
```
Comment (gaetanmaisse) [inline stories/lib/utils.test.js:89]: `string arrays` -> 🔍 in the test we are using number arrays

File: src/molecules/gv-row-expandable.js [hunk 3]
```diff
@@ -14,50 +14,72 @@
...
 
 /**
  * Expandable row
  *
+ * @fires gv-expandable-row:toggle - When user toggle row
+ *
  * @attr {Boolean} open - When true, opens details by default
+ * @attr {String} icon - An optional icon to show open feature
+ * @attr {String} icon-opened -  An optional icon to show close feature
+ * @attr {Boolean} only-summary - When true, only summary is clickable for toggle row
```
Comment (gaetanmaisse) [inline src/molecules/gv-row-expandable.js:29]: `only-summary` isn't straight forward to understand maybe we should rename it with something with the opposite meaning: Collapse on details clicked --- When false, only summary is clickable to collapse the row

But I didn't find an easy name 😞

File: src/molecules/gv-row-expandable.js [hunk 4]
```diff
@@ -71,30 +93,31 @@ export class GvRowExpandable extends LitElement {
...
   }
 
-  _onClick() {
+  _onClick(event) {
+    if (this.onlySummary) {
+      const summary = this.shadowRoot.querySelector('.summary');
+      if (!summary.contains(event.target)) {
+        return;
+      }
+    }
```
Comment (gaetanmaisse) [inline src/molecules/gv-row-expandable.js:107]: It is causing troubles when set to true, a click on the `summary` slot is filtered too

**Review by jgiovaresco — APPROVED** [review]
LGTM

**Review by aelamrani — APPROVED** [review]

**Pull request merged**